docs/run.sh

```diff
@@ -68,6 +68,8 @@ ShowBanner() {
     echo "Current value of the (overrideable) R API variable 'R_VERSION': ${R_VERSION}"
     echo "Current Ubuntu distribution per 'lsb_release': '$(lsb_release -ds)' aka '$(lsb_release -cs)'"
     echo ""
+    echo "Current coverage type per 'COVERAGE_TYPE': ${COVERAGE_TYPE}"
```

Comment:
Hm. Not sure we want to always show it.

Reply:
Not sure it is worth removing either though so no rush.  I probably need a function 'verbose' or 'showSettings' that displays these.